packages/@n8n/nodes-langchain/nodes/llms/LMOpenAi/LmOpenAi.node.ts

```diff
@@ -189,10 +169,45 @@ export class LmOpenAi implements INodeType {
 		],
 	};
 
+	methods = {
+		listSearch: {
+			async openAiModelSearch(this: ILoadOptionsFunctions) {
+				const results = [];
+
+				const options = this.getNodeParameter('options', {}) as IDataObject;
+
+				let uri = 'https://api.openai.com/v1/models';
+
+				if (options.baseURL) {
+					uri = `${options.baseURL}/models`;
+				}
+
+				const { data } = await this.helpers.requestWithAuthentication.call(this, 'openAiApi', {
+					method: 'GET',
```

Comment:
I see some warnings here.
![CleanShot 2023-11-27 at 15 33 24](https://github.com/n8n-io/n8n/assets/12657221/a8131d5e-fc4d-4179-abb4-e86469691427)
To get rid of them, can we define options as interface and asert it both on L211 and L177 instead of `IDataObject`